wpapi-class-shortcode.php

```diff
@@ -54,7 +54,7 @@ class WpapiShortcodes
 
     function set_query($attr){
         extract(shortcode_atts($this->default, $attr));
-        $q = "filter[orderby]={$orderby}";
+        $q = "&filter[orderby]={$orderby}";

```

Comment:
__[phpmd]__
* Avoid variables with short names like $q. Configured minimum length is 3.
* Avoid unused local variables such as '$orderby'.